### ISSUE TITLE
Timeline with broken structure (css issue?)

### ISSUE THREAD

**Issue by henrique-voni** [CodyHouse/horizontal-timeline#11]
Hello, 

I'm using your timeline as a Wordpress plugin, but the timeline is being shown unformatted. 

See: https://imgur.com/a/HS58muW

The site is on a local environment. If more info is needed, then i can create a repo for the plugin.

Thanks,

Henrique

**Comment by anndro** [comment]
I guess its normal behavior of the code. I fixed with adding null div before event div.

```
<div class="events-wrapper">
    <div style="background: #dfdfdf;top: 49px; height: 2px; position: relative;"></div>
    <div class="events">
    </div>
</div>
```

**Comment by claudia-romano** [comment]
Hi there, sorry for getting back after quite some time.
We have just released a new version (2.0) of this experiment (based on the [CodyHouse framework](https://codyhouse.co/ds/docs/framework)).

It looks like you were not importing the reset style. If you are using v2.0, make sure to import the framework and you should not experience this issue anymore.
